code/modules/mob/holder.dm

```diff
@@ -592,6 +592,22 @@ var/list/holder_mob_icon_cache = list()
 /obj/item/weapon/holder/pai/rabbit
 	icon_state = "rabbit_rest"
 	item_state = "rabbit"
+/obj/item/weapon/holder/pai/custom
+	var/spriteloaded = 0
+
+/obj/item/weapon/holder/pai/custom/process()
+
+	set_paiholder()
```

Comment:
This is. A really. Bad idea. Why are we doing this in a process loop...?

Reply:
Agreed, this shouldn't be here